### ISSUE TITLE
ghcr.io/tinglesoftware/dependabot-updater:0.17 image is missing

### ISSUE THREAD

**Issue by Mahagon** [tinglesoftware/dependabot-azure-devops#638]
Hi,

since this morning (see timestamps in logs) we can't pull the "ghcr.io/tinglesoftware/dependabot-updater:0.17" image anymore, it seems to be missing:

`curl -H "Authorization: Bearer ${TOKEN}" https://ghcr.io/v2/tinglesoftware/dependabot-updater/tags/list`

`{"name":"tinglesoftware/dependabot-updater","tags":["latest","63d68f5","0.17.2-ci0023","4971684","0.17.2-ci0024","a95a9be","0.17.2-ci0025","9963ce8","0.17.2-ci0026","92eab25","0.17.2-ci0027","3dc5006","0.17.2-ci0028","bb44a33","0.17.2-ci0029","4890856","0.17.2-ci0030","4242944","0.17.2-ci0031","25d9a60","0.17.2-ci0032","b7a24ac","0.17.2-ci0033","0d10c46","0.17.2-ci0034","00320eb","0.17.2-ci0035","c424a9b","0.17.2-ci0037","3806ebe","0.17.2-ci0038","98bb05c","0.17.2-ci0039","9be3cd2","0.17.2-ci0040","043abba","0.17.2-ci0041","68487b8","0.17.2-ci0042","8de911f","0.17.2-ci0044","f63e7c6","0.17.2-ci0046","f651a6b","0.17.2-ci0045","b10e721","0.17.2-ci0047","f9494ba","0.17.2-ci0048","29078f6","0.17.2-ci0050","aeed9fc","0.17.2-ci0049"]}`

is there a cleanup task running?

```
2023-05-22T07:08:36.1047344Z ##[section]Starting: Dependabot
2023-05-22T07:08:36.1051771Z ==============================================================================
2023-05-22T07:08:36.1051880Z Task         : Dependabot
2023-05-22T07:08:36.1051932Z Description  : Automatically update dependencies and vulnerabilities in your code
2023-05-22T07:08:36.1052025Z Version      : 1.17.489
2023-05-22T07:08:36.1052072Z Author       : Tingle Software
2023-05-22T07:08:36.1052122Z Help         : For help please visit https://github.com/tinglesoftware/dependabot-azure-devops
2023-05-22T07:08:36.1052220Z ==============================================================================
2023-05-22T07:08:36.3277032Z [command]/usr/bin/docker run --rm -i -e GITHUB_ACCESS_TOKEN=*** -e DEPENDABOT_PACKAGE_MANAGER=docker -e DEPENDABOT_OPEN_PULL_REQUESTS_LIMIT=5 -e DEPENDABOT_DIRECTORY=images/eventhub-loki-bridge -e DEPENDABOT_TARGET_BRANCH=master -e DEPENDABOT_FAIL_ON_EXCEPTION=true -e DEPENDABOT_SKIP_PULL_REQUESTS=true -e AZURE_ORGANIZATION=REDACTED -e AZURE_PROJECT=REDACTED -e AZURE_REPOSITORY=REDACTED -e AZURE_ACCESS_TOKEN=*** -e AZURE_MERGE_STRATEGY=rebase ghcr.io/tinglesoftware/dependabot-updater:0.17
2023-05-22T07:08:36.4516450Z Unable to find image 'ghcr.io/tinglesoftware/dependabot-updater:0.17' locally
2023-05-22T07:08:37.0823755Z docker: Error response from daemon: manifest unknown.
2023-05-22T07:08:37.0827280Z See 'docker run --help'.
2023-05-22T07:08:37.0914009Z ##[error]The process '/usr/bin/docker' failed with exit code 125
2023-05-22T07:08:37.0955004Z ##[section]Finishing: Dependabot
```

**Comment by drvup** [comment]
Same issue on my side

**Comment by pir264** [comment]
Same here

**Comment by mburumaxwell** [comment]
Apologies to everyone in the dark. This is now fixed, with new version `1.18.*` or `0.18.*`
